tests/foreman/ui/test_errata.py

```diff
@@ -61,17 +64,27 @@ def _generate_errata_applicability(hostname):
     host.errata_applicability(synchronous=False)
 
 
-def _install_client_package(client, package, errata_applicability=False):
+def _install_client_package(client, package):
     """Install a package in virtual machine client.
+    Errata applicability regenerated after successful yum execution.
```

Comment:
Without errata applicability check here, why we keeping functions for just single install and remove tasks, can't we get rid of them instead?